request-manager/spec/parser.go

```diff
@@ -12,39 +12,33 @@ import (
 )
 
 // Parse a single request (YAML) file.
-// 'logFunc' is a Printf-like function used to log warning(s) should they occur.
-// Errors are returned, not logged.
-func ParseSpec(specFile string, logFunc func(string, ...interface{})) (Specs, error) {
-	var spec Specs
-
+func ParseSpec(specFile string) (spec Specs, err error, warning error) {
```

Comment:
In Go, `error` is pretty much always the last return arg:
```go
ParseSpec(specFile string) (spec Specs, warning error, err error)
```
Normally, don't use named args unless there's an explicit reason for doing so, but this maybe a case with a reason: two return `error` is confusing unless they're named.